packages/amplify-prompts/src/progressbars/multibar.ts

```diff
@@ -14,21 +14,21 @@ import {
  * Abstraction to create multiple progress bars inside one re writable block.
  */
 export class MultiProgressBar {
-    private count : number;
-    private terminal : Terminal;
-    private options : BarOptions;
-    private bars : {name: string, bar: Bar}[];
+    private count: number;
+    private terminal: Terminal;
+    private options: BarOptions;
+    private bars: { name: string, bar: Bar }[];
     private lastDrawnTime: number;
     isActive: boolean;
     private refreshRate: number;
-    private frameCount : number;
-    private frames : string[];
+    private frameCount: number;
+    private frames: string[];
     private timer!: ReturnType<typeof setTimeout>;
-    private prefixText : string;
+    private prefixText: string; // Header text goes in front of an animated spinner
```

Comment:
should we call the field `header` then ?

Reply:
Added to backlog refactor task